includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -169,14 +169,17 @@ public function register_mark_tasks_complete_route() {
 	 * @return array Setup Wizard data
 	 */
 	public function get_data() {
+		$show_extensions = current_user_can( 'activate_plugins' ) && current_user_can( 'update_plugins' );
+
 		return [
-			'tasks'        => $this->tasks_provider->get(),
-			'quick_links'  => $this->quick_links_provider->get(),
-			'help'         => $this->help_provider->get(),
-			'guides'       => $this->guides_provider->get(),
-			'news'         => $this->news_provider->get(),
-			'promo_banner' => $this->promo_banner_provider->get(),
-			'notices'      => $this->notices_provider->get(),
+			'tasks'           => $this->tasks_provider->get(),
+			'quick_links'     => $this->quick_links_provider->get(),
+			'help'            => $this->help_provider->get(),
+			'guides'          => $this->guides_provider->get(),
+			'news'            => $this->news_provider->get(),
+			'promo_banner'    => $this->promo_banner_provider->get(),
+			'notices'         => $this->notices_provider->get(),
+			'show_extensions' => $show_extensions,
```

Comment:
I'm purposefully not allowing this to be filtered so I thought we could avoid any kind of provider. Happy to reevaluate if people feel strongly.